vllm/config.py

```diff
@@ -1028,6 +1035,10 @@ def _verify_args(self) -> None:
     def is_multi_step(self) -> bool:
         return self.num_scheduler_steps > 1
 
+    @property
+    def is_multi_step_chunked_prefill(self) -> bool:
+        return self.is_multi_step and self.chunked_prefill_enabled
```

Comment:
If possible I don't want to introduce this property.

Reply:
It can be removed. I checked that the PR uses it 5 please in 2 files. I can get rid of it.